### ISSUE TITLE
Issue Report - Output Issue (Other)

### ISSUE THREAD

**Issue by OswaldMurphy** [ezfe/tellraw#159]
Please enter steps to reproduce the issue below, as well as any other information you want to include
I am in 1.9 and it is just not working out...

 Provided Data - Do not modify below this line

```
[{"text":"Click this!","color":"blue","bold":true,"italic":true,"underlined":true,"clickEvent":{"action":"open_url","value":"paypal.me/OswaldMurphy"},"hoverEvent":{"action":"show_text","value":{"text":"","extra":[{"text":"CLICK","color":"green"}]}}}]
```

**Comment by ezfe** [comment]
Are you having issues with the link clicking?

**Comment by OswaldMurphy** [comment]
Yes. I can not actually click the link. I made my own small line of code (i'm a developer) to try to fix it for 1.9. It just wasn't doing anything. It didn't even say something in chat... I did though get it to say something in my other code but I could not click it.

**Comment by ezfe** [comment]
Ok, use the Import button and reimport this text

```json
{"command":"/tellraw @p %s","jobject":[{"text":"Click this!","color":"blue","bold":true,"italic":true,"underlined":true,"clickEvent":{"action":"open_url","value":"http://paypal.me/OswaldMurphy"},"hoverEvent":{"action":"show_text","value":{"text":"","extra":[{"text":"CLICK","color":"green"}]}}}],"jtemplate":"tell raw"}
```

The issue was that you need to include http:// or https:// in front of the URL
